### ISSUE TITLE
Ask what the specific hyperparameter Settings are during your training

### ISSUE THREAD

**Issue by Laulen** [luluyuu/COMO#7]
May I ask what the specific hyperparameter Settings are during your training? For example, batch size, learning rate, etc. It was not specifically stated in your journal article. I'm very sorry that I encountered problems when reproducing your excellent performance. I have successfully installed the environment and run the code, and trained according to the 150epochs in your paper, but the results are far from satisfactory. So I want to know your hyperparameter Settings.

And the number of parameters is also quite different from that in the paper. Perhaps you could provide screenshots of the experiment you saved during the process to prove that I'm wrong?

<img width="1251" height="934" alt="Image" src="https://github.com/user-attachments/assets/320fa122-b965-4cf4-9338-4dab02ab8cb2" />

<img width="1138" height="567" alt="Image" src="https://github.com/user-attachments/assets/6483bb96-3905-4ba3-887e-cd85a2196b11" />

**Comment by luluyuu** [comment]
> > Thank you very much for your interest in this project. Regarding your experimental results, they appear to be unusually low. Please first check whether the data are correct and consistent with the provided dataset. You can also inspect the visualized outputs in the generated files. I have cropped the white borders of the DroneVehicle dataset, so please ensure that the labels correspond correctly to the images.
> > As for the hyperparameters you mentioned, I did not perform extensive tuning. Most of the parameters can be found in ./ultralytics/cfg/default.yaml. The learning rate is adjusted based on the optimizer: 0.01 for SGD, and 0.001 for AdamW.
> > Regarding the params issue, my method follows the same computation approach as SuperYOLO, as shown in the figure below — it’s a fairly straightforward calculation.
> > If the results are still abnormal after the above checks, please try reconfiguring your environment. Both causal-conv1d and mamba-ssm were downloaded from their respective GitHub repositories at specific versions, and I replaced the corresponding files accordingly.
> > Finally, here are some of my log results.
> > <img alt="Image" width="1243" height="869" src="https://private-user-images.githubusercontent.com/167509197/499360746-79d7e0a0-01c5-49cd-acb3-73bcf474b67e.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjA2MTQ3NzYsIm5iZiI6MTc2MDYxNDQ3NiwicGF0aCI6Ii8xNjc1MDkxOTcvNDk5MzYwNzQ2LTc5ZDdlMGEwLTAxYzUtNDljZC1hY2IzLTczYmNmNDc0YjY3ZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAxNlQxMTM0MzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wY2M1MDQxY2ZhOGE0Yzc5MWI2MDk5YWQ2NWYzYzFhYTMzNWVlZDBlZTRhYzdjYmZjMDRkOTY3MDU4MjJhNzYxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.oRf4oExB4tQRWM5EPoA5VjXq0xVu5UGJsETY8rt9HjI"> <img alt="Image" width="1281" height="284" src="https://private-user-images.githubusercontent.com/167509197/499361329-a1e8178c-e3cb-4dd9-9772-2c7165a13d0b.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjA2MTQ3NzYsIm5iZiI6MTc2MDYxNDQ3NiwicGF0aCI6Ii8xNjc1MDkxOTcvNDk5MzYxMzI5LWExZTgxNzhjLWUzY2ItNGRkOS05NzcyLTJjNzE2NWExM2QwYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAxNlQxMTM0MzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01MWY1YTZjOTAyZGMzMmQ3MjJlODk2OTU1ZDU1MGE1MDYxNjJkZmE4Yjc2MDY5NDEyZDg5ZTdhNDdlMGJhMzhjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.HT01as3cFZ0azvq8uB-rIDw9rga3tYLz6uRRGXFOGbs"> <img alt="Image" width="1044" height="232" src="https://private-user-images.githubusercontent.com/167509197/499361787-f01ca3bc-8e72-43f6-9b26-769860708d5c.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjA2MTQ3NzYsIm5iZiI6MTc2MDYxNDQ3NiwicGF0aCI6Ii8xNjc1MDkxOTcvNDk5MzYxNzg3LWYwMWNhM2JjLThlNzItNDNmNi05YjI2LTc2OTg2MDcwOGQ1Yy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUxMDE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MTAxNlQxMTM0MzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02YmZlNjZkMTRlNzA2M2ZlMjRmMTU2N2FiNTRjMjlkNTlmZmIxYTg5MGNmMWZhNTE4ZTU4MjRjNDg3ZmU3ZjdkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.ECC1qYY00RegtqZt_beZy_7PpfJuaCEoTwrRhMZcb8c">
> 
> Currently, there is still a discrepancy between the parameters I actually obtained and those in the your paper. Perhaps you could share your best.pt file? I want to directly test whether it's due to code issues or machine differences.
> 
> <img alt="Image" width="718" height="254" src="https://private-user-images.githubusercontent.com/53033753/501996232-720a972e-8cb5-443a-8158-576099d13aa4.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjE2NTQ4NzcsIm5iZiI6MTc2MTY1NDU3NywicGF0aCI6Ii81MzAzMzc1My81MDE5OTYyMzItNzIwYTk3MmUtOGNiNS00NDNhLTgxNTgtNTc2MDk5ZDEzYWE0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTEwMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMDI4VDEyMjkzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTFiNGQxY2I0MzVmM2I4YWNlMDQ5ZmY1MDY4NjI2NjllNWQxYjQxN2ZlZTc5OTZhMGY2YWYwNjhiMTI3NGNhMDkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.xGdNbFZm3ne7vnCVZuuRQgerJLA1WrxwMxPQWipi_i0"> <img alt="Image" width="477" height="250" src="https://private-user-images.githubusercontent.com/53033753/501996233-9745723e-6593-4b85-a9d8-8ccdede5511a.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjE2NTQ4NzcsIm5iZiI6MTc2MTY1NDU3NywicGF0aCI6Ii81MzAzMzc1My81MDE5OTYyMzMtOTc0NTcyM2UtNjU5My00Yjg1LWE5ZDgtOGNjZGVkZTU1MTFhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTEwMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMDI4VDEyMjkzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQwYjExMWU1NGFlMDA5ZDc1ZWUzZmEyZmI0MGJiOGFlOWM5YmNlZDJkN2U1ZmNiNmQ1YjFkYzNhN2FjZmRiZDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.6HNirSkVZ6jx_hWn8JPodkreqhOPzaHbC6pENhaxpMI"> <img alt="Image" width="942" height="209" src="https://private-user-images.githubusercontent.com/53033753/501997532-a869c4a1-73c2-43d5-ba28-f5388af7af4d.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjE2NTQ4NzcsIm5iZiI6MTc2MTY1NDU3NywicGF0aCI6Ii81MzAzMzc1My81MDE5OTc1MzItYTg2OWM0YTEtNzNjMi00M2Q1LWJhMjgtZjUzODhhZjdhZjRkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTEwMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMDI4VDEyMjkzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRmOTAzYjU5NjNlYzFiOWFiMTYwMDYzNDY4Y2JiMDFlZTM3ZjNiYTZmMWRkNjJlNmE5M2Y4ZTY3M2E0YTBkYzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.JCPzUJkI8xQn-7iQWmQcGFmFkdc34JJ2dP5YP0189Pg">

We greatly appreciate your continued attention. We’re truly sorry that the previous repository had many issues. The checkpoint and the more complete repository have been uploaded now. Please re-download and try again.
